styleguide/derek/examples/_solutions.html

```diff
@@ -1,707 +1,600 @@
-<div class="container">
+
+
+
+<!-- main content wrap -->
+<div id="content-wrap" class="styleguide">
```

Comment:
We can leave out the id and class here. The panels content type template file will contain this tag so we can avoid bringing these over from the legacy markup.